backend/src/main/java/com/woowacourse/kkogkkog/domain/CouponEvent.java

```diff
@@ -6,11 +6,12 @@
 
 public enum CouponEvent {
 
+    INIT(CouponEvent::canInit),
     REQUEST(CouponEvent::canRequest),
     CANCEL(CouponEvent::canCancel),
     DECLINE(CouponEvent::canDecline),
     ACCEPT(CouponEvent::canAccept),
-    FINISH(CouponEvent::canFinish),
+    FINISH(CouponEvent::canFinish)
```

Comment:
```suggestion
    FINISH(CouponEvent::canFinish),
```
예전에 이야기 했던대로 Enum 필드들 마지막은 콤마로 맞추는건 어떤까요?